app/views/layouts/snowcrash/_notifications.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="dropdown-menu">
```

Comment:
As mentioned above, we could render the whole dropdown server side and use very little js in `index.js.erb`.

Then we should also use cache with the user and notifications in the key.